domains/Accounts/Tests/Feature/UserCreateTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Domains\Accounts\Tests\Feature;
+
+use Domains\Accounts\Models\User;
+use Domains\Accounts\Notifications\VerifyEmailNotification;
+use Faker\Factory;
+use Faker\Generator;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Notification;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class UserCreateTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    protected Generator $faker;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->faker = Factory::create();
+        Notification::fake();
+    }
+
+    /** @test */
+    public function it_fails_to_create_a_user_on_validation_errors(): void
+    {
+        $this->post(route('accounts.store'), [])
+            ->seeJsonStructure([
+                'name',
+                'email',
+                'password',
+            ])->assertResponseStatus(Response::HTTP_UNPROCESSABLE_ENTITY);
```

Comment:
Nit: care to break this call into its own line?

Reply:
done